tests/pages/components/task-details.js

```diff
@@ -22,7 +22,10 @@ export default {
     hasStrongText: isVisible('strong'),
     text: text()
   },
-
+  nullCommentBody:{
+    scope: '.null-comment-body',
```

Comment:
I would rename this class to `data-test-null-body` (in all cases here) which will get automatically removed later. Also think slightly less confusing than `comment` in context of task.

Reply:
Whoops. So this is not a CSS class I'm looking for but a `data-` attr. Just to clarify.

Reply:
@JoshSmith You sure you don't want a class to, for potential styling?

Github, for example, seems to italicize the text and make it a bit lighter than the normal body text.

Reply:
To verify, is this what you are looking for?

```<em class="null-comment-body" data-test-null-body=true>No description provided.</em>```

Reply:
`<em data-test-null-body>No description provided.</em>`

Reply:
In `/styles/_markdown.scss` I would add

```css
.markdown-body--empty {
  em {
    color: $text--light;
  }
}
```